DEPS

```diff
@@ -30,6 +30,11 @@ vars = {
   # for the web engine.
   'download_emsdk': False,
 
+  # For experimental features some dependencies may only be avaialable in the master/main
+  # channels. This variable is being set when CI is checking out the repository.
+  'release_candidate': False,
```

Comment:
I wonder if it would be better to have a "channel" variable, it's likely that down the road Windows ARM64 and RISCV SDKs will be on beta but not on stable.

Reply:
+1

Reply:
I can change the variable name however we don't have a good way to detect the channel. Flutter builds everything in release candidate branches and for new beta|stable releases we only know the channel when we push the release candidate branch to the channel branch.

Would it be confusing naming the variable channel and always setting to beta?

Reply:
Hmm, if it's only ever beta, it wouldn't help much. How would we know to have Windows ARM64 on beta but not on stable. It would be possible to determine what Dart channel is to be used from the Dart version. Would it help if the variable was `dart_channel`? We always CP a Dart release, and when that CP happens, maybe we can change the value of `dart_channel` to match the CP'ed version of Dart?

Reply:
Not sure how can we get that information without hardcoding it somewhere in the infra configs, any ideas?